java/io/bazel/rules/closure/webfiles/server/WebfilesServer.java

```diff
@@ -114,7 +118,14 @@ public HostAndPort spawn() throws InterruptedException {
       while (actualAddress == null) {
         wait();
       }
-      return actualAddress;
+      return NetworkUtils.createUrlAddress(actualAddress);
```

Comment:
Looks like you can actually update actualAddress assignment to do the NetworkUtils.createUrlAddress call and remove this one and the other call below.